Container/Container.cs

```diff
@@ -1,57 +1,33 @@
-﻿using System.ComponentModel.DataAnnotations;
-
-namespace Container
+﻿namespace Container
 {
     public class Container
     {
-        List<int> container = new List<int>();
-        public int maxLength = 5;
+        private readonly List<int> _container = new(capacity: 5);
```

Comment:
Zazwyczaj zawsze wskazuje się poziom enkspsulacji, tutaj dodałem też `readonly`, bo nie przewiduję, żeby była kiedykolwiek potrzeby nadpisania tej listy.
Dodałem `capacity: 5` do konstruktora listy, co jest pewną niewielką optymalizację. W naszym przykładzie nie ma to oczywiście żadnego znaczenia, ale w prawdziwych aplikacjach warto to stosować. Dzięki temu .NET stworzy pod spodem miejsce w pamięci dla dokładnie 5 intów, nie więcej.

Reply:
pola prywatne w klasach zazwyczaj prefixuje się `_`